o2a/o2a_libs/README.md

```diff
@@ -0,0 +1,3 @@
+# o2a Library
+
+This is an additional library for Oozie To Airflow migration tool
```

Comment:
Please add to the description that importing it is required by the Airflow DAGs generated by the tool.

Reply:
@michalmodras I have added